clang/lib/Basic/Targets/Mips.cpp

```diff
@@ -273,6 +273,34 @@ bool MipsTargetInfo::validateTarget(DiagnosticsEngine &Diags) const {
     Diags.Report(diag::err_mips_fp64_req) << "-mfp64";
     return false;
   }
+  // FPXX requires mips2+
+  if (FPMode == FPXX && CPU == "mips1") {
+    Diags.Report(diag::err_opt_not_valid_with_opt) << "-mfpxx" << CPU;
```

Comment:
Here, it depends on: https://github.com/llvm/llvm-project/pull/92122
Otherwise  `clang -cc1 -triple mips -target-cpu mips1` will always fail.